plugins/modules/load_balancer_health_monitor.py

```diff
@@ -0,0 +1,442 @@
+  type: int
+  sample: 2
+timeout_s:
+  description: The maximum time allowed for an individual check in seconds
+  returned: success when not state == absent
+  type: int
+  sample: 1
+up_threshold:
+  description: The number of checks that need to be successful before the monitor_status of a pool member changes to "up"
+  returned: success when not state == absent
+  type: int
+  sample: 2
+down_threshold:
+  description: The number of checks that need to fail before the monitor_status of a pool member changes to "down"
+  returned: success when not state == absent
+  type: int
+  sample: 3
+type:
+  description: The type of the health monitor
+  returned: success when not state == absent
+  type: str
+http:
+  description: Advanced options for health monitors with type "http" or "https"
+  returned: success when not state == absent
+  type: dict
+  sample: [ {
+                "expected_codes": [
+                    "200"
+                ],
+                "method": "GET",
+                "url_path": "/",
+                "version": "1.0",
+                "host": null
+            } ]
+tags:
+  description: Tags assosiated with the load balancer
+  returned: success
+  type: dict
+  sample: { 'project': 'my project' }
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ..module_utils.api import (
+    AnsibleCloudscaleBase,
+    cloudscale_argument_spec,
+)
+
+ALLOWED_STATES = ('present',
+                  'absent',
+                  )
+ALLOWED_HTTP_POST_PARAMS = ('expected_codes',
+                            'host',
+                            'method',
+                            'url_path')
+
+
+class AnsibleCloudscaleLoadBalancerHealthMonitor(AnsibleCloudscaleBase):
+
+    def __init__(self, module):
+        super(AnsibleCloudscaleLoadBalancerHealthMonitor, self).__init__(
+            module,
+            resource_name='load-balancers/health-monitors',
+            resource_key_name='pool',
+            resource_create_param_keys=[
+                'pool',
+                'timeout_s',
+                'up_threshold',
+                'down_threshold',
+                'type',
+                'http',
+                'tags',
+            ],
+            resource_update_param_keys=[
+                'delay_s',
+                'timeout_s',
+                'up_threshold',
+                'down_threshold',
+                'expected_codes',
+                'http',
+                'tags',
+            ],
+        )
+
+    def query(self):
+        # Initialize
+        self._resource_data = self.init_resource()
+
+        resource_key_pool = 'pool'
+        uuid = self._module.params[self.resource_key_uuid]
+        pool = self._module.params[resource_key_pool]
+        matching = []
+
+        # Either search by given health monitor's UUID or
+        # search the health monitor by its acossiated pool UUID (1:1)
+        if uuid is not None:
+            super().query()
```

Comment:
Does this work with all supported Python versions?

Reply:
According to https://stackoverflow.com/a/33469090 it needs to be changed to support python 2.x (and ansible 2.9).

Reply:
When I try to follow the suggestion above, I get the following error:
```
ERROR: plugins/modules/load_balancer.py:194:8: bad-super-call: Bad first argument 'AnsibleCloudscaleLoadBalancer' given to super()
ERROR: plugins/modules/load_balancer_health_monitor.py:267:12: bad-super-call: Bad first argument 'AnsibleCloudscaleLoadBalancerHealthMonitor' give
n to super()
```

I'm going to revert the change again.